src/translations/de.json

```diff
@@ -10,9 +10,10 @@
   },
   "source": {
     "silent": "Leise",
-    "standard": "Normal",
+    "standard": "Balanciert",
```

Comment:
I would prefer to leave it as it is or change it to "Standard"

Reply:
Please, consider this comment.